### PR TITLE
cplp-1168 - fix for app user response and swagger documentation

### PR DESCRIPTION
**Pull request by Akshay-Chaudhari-BMW**
fix for app user response issue and swagger documentation has been added.

### CODE REVIEW

File: src/administration/CatenaX.NetworkServices.Administration.Service/Controllers/UserController.cs [hunk 1]
```diff
@@ -113,9 +113,19 @@ public UserController(ILogger<UserController> logger, IUserBusinessLogic logic)
...
 
+         /// <summary>
+        /// Get company app users by appId
+        /// </summary>
+        /// <param name="appId">Get company app users by appId</param>
+        /// <param name="page">page index start from 0</param>
+        /// <param name="size">size to get number of records</param>
+        /// <returns>Company App Users Details</returns>
+        /// <remarks>Example: GET: /api/administration/user/owncompany/apps/5cf74ef8-e0b7-4984-a872-474828beb5d3/users?page=0&size=15</remarks>
+        /// <response code="200">Result as a Company App Users Details</response>
```
Comment (jjeroch) [inline src/administration/CatenaX.NetworkServices.Administration.Service/Controllers/UserController.cs:124]: Lets change the message to "Returns the company users with assigned role for the requested app id." eventhough we still need to do small adjustments on the API until this is working ;)

**Comment by jjeroch** [comment]
I can take the review, we can release Norbert and Evelyn for the more critical topics :)

File: src/administration/CatenaX.NetworkServices.Administration.Service/Controllers/UserController.cs (hunk 1 above)
Comment (jjeroch) [reply src/administration/CatenaX.NetworkServices.Administration.Service/Controllers/UserController.cs:124]: @Akshay-Chaudhari-BMW  can you please make the small change, the rest is all good

**Review by jjeroch — CHANGES_REQUESTED** [review]

File: src/administration/CatenaX.NetworkServices.Administration.Service/Controllers/UserController.cs (hunk 1 above)
Comment (Akshay-Chaudhari-BMW) [reply src/administration/CatenaX.NetworkServices.Administration.Service/Controllers/UserController.cs:124]: resolved with msg change

**Review by jjeroch — APPROVED** [review]

**Comment by jjeroch** [comment]
@Akshay-Chaudhari-BMW ready to merge whenever you want ;)

**Comment by Akshay-Chaudhari-BMW** [comment]
We can merge it now.

**Pull request merged**